jenkins/root-build.cmake

```diff
@@ -476,6 +476,13 @@ function(GET_ALL_SUPPORTED_MODULES_LINUX)
     EXPORT_CTEST_ENVVAR(ROOTTEST_IGNORE_NUMBA_PY3)
   endif()
 
+  # Fedora 32 upwards have python2 completely deprecated, also installation of python2-pip is
+  # not possible via dnf.
+  # Note that we veto below all fedora version that NOT match the regex.
+  if(NOT "${LABEL}" MATCHES "fedora(29|30|31)")
```

Comment:
That's too aggressive - this will veto numba e.g. for all Ubuntus and our Debian machine. Could you instead be specific on what to exclude it on?

Reply:
You are right, the logic is totally broken ... I'll just exclude for `fedora32`.

Reply:
Fixed!